src/pages/Lectures.tsx

```diff
@@ -0,0 +1,198 @@
+                break;
+        }
+        dateString += " " + date.getFullYear() + " - ";
+        switch (date.getDay()) {
+            case 1:
+                dateString += "Montag";
+                break;
+            case 2:
+                dateString += "Dienstag";
+                break;
+            case 3:
+                dateString += "Mittwoch";
+                break;
+            case 4:
+                dateString += "Donnerstag";
+                break;
+            case 5:
+                dateString += "Freitag";
+                break;
+            case 6:
+                dateString += "Samstag";
+                break;
+            case 7:
+                dateString += "Sonntag";
+                break;
+        }
+        return dateString
+    }
+
+    //states for refresh button and lectures
+    const [lectures, setLectures] = useState([] as LectureGrouped[]);
+    const [refreshing, setRefresh] = useState(false);
+
+
+    useEffect(() => {
+        getData('lectures').then((lectureList) => {
+            if (lectureList) {
+                const lect: Lecture[] = lectureList.map((lecture: Lecture) => {
+                    const lec: Lecture = {
+                        id: lecture["id"],
+                        date: new Date(lecture["date"]),
+                        startTime: new Date(lecture["startTime"]),
+                        endTime: new Date(lecture["endTime"]),
+                        name: lecture["name"],
+                        type: lecture["type"],
+                        rooms: lecture["rooms"],
+                        course: lecture["course"]
+                    }
+                    return lec;
+                });
+                setLectures(groupList(lect));
+
+            }
+
+        })
+    }, [])
+    //on refresh it gets the course
+    // if it is not connected to the internet it gets an update that it is not connected and it returns it to the user
+    const refresh = useCallback(() => {
+        setRefresh(true);
+        NetInfo.fetch().then(netstate=>{
+            if (netstate.isConnected) {
+                NetworkService.getLectures('MOS-TINF21A').then((lecture: Lecture[] | null) => {
+                    if (lecture) {
+                        setLectures(groupList(lecture));
+                        setRefresh(false);
+                    }
+                });
+            } else {
+                let toast = Toast.show("Sie sind nicht zum Internet verbunden")
+                setRefresh(false)
+            }
+        })
+
+    }, []);
+
+
+    return (
+        <ScrollView style={scrollViewStyle.scrollView}
+                    refreshControl={
+                        <RefreshControl refreshing={refreshing}
+                                        onRefresh={refresh}
+                        />}
+                    ref={ref}
+        >
+            {lectures.map((lectureGrouped) => {
+                return (
+                    <View>
```

Comment:
Bitte versuch globale styles aus dem Stylecomponent zu verwenden soweit es möglich ist

Reply:
also soll ich das in nem globane stil einbauen oder wat

Reply:
In den stylecomponent den ich geschrieben hatte ja